daras_ai_v2/base.py

```diff
@@ -367,30 +367,34 @@ def _render_header(self):
         can_save = self.can_user_save_run(sr, pr)
         request_changed = self._has_request_changed()
 
-        with gui.div(className="d-flex justify-content-between mt-4"):
-            with gui.div(className="d-lg-flex d-block align-items-center"):
-                if not tbreadcrumbs.has_breadcrumbs() and not self.current_sr_user:
-                    self._render_title(tbreadcrumbs.h1_title)
-
-                if tbreadcrumbs:
```

Comment:
the extra heading was a regression. this if condition was busted - `tbreadcrumbs.has_breadcrumbs` needs to be used.